index.bs

```diff
@@ -306,14 +309,42 @@ partial interface XRSession {
   readonly attribute XRDepthUsage depthUsage;
   readonly attribute XRDepthDataFormat depthDataFormat;
   readonly attribute XRDepthType? depthType;
+  readonly attribute boolean? depthActive;
+
+  undefined startDepthSensing();
+  undefined stopDepthSensing();
```

Comment:
After reading through the change, I have a _very_ minor preference for the verbiage `pause` and `resume` over `start` and `stop`.

The logic being that requesting the session with the depth sensing feature was the "start" signal, and the user cannot enable the feature after the session has begun. They can only temporarily pause it, and you can only resume something if it was started in the first place.

This is an extremely picky verbiage nit, and so I won't take offense if you simply prefer `start` and `stop`. :grin: Just wanted to leave my two cents.